src/pages/settings/tabs/currencies/edit-currency.vue

```diff
@@ -123,10 +142,38 @@ export default defineComponent({
       },
     );
 
+    onMounted(() => {
+      isChecked.value = props.currency.custom
+    })
+
     const onDeleteHandler = () => {
       emit('delete');
     };
 
+    const deleteExchangeRates = async () => {
+      try {
+          deleteCustomRate([
+            {
+              baseCode: props.currency.code,
+              quoteCode: props.currency.quoteCode,
+            },
+            {
+              baseCode: props.currency.quoteCode,
+              quoteCode: props.currency.code,
+            },
+          ]),
+        emit('submit');
+
```

Comment:
1. Formatting is broken
2. No `await` for `deleteCustomRate` so if there will be any error, it won't be caught